CMSgov/easi-app
# [NOREF] Create attendee by default

# NOREF

## Changes and Description

- Make `component` and `role` nullable on TRB Attendees
- Create an Attendee when making a TRB Request with the caller's EUA ID & info
- TODO: (@aterstriep @adamodd?) Frontend changes to remove code that creates a new attendee for the requester on the "Attendees" page of the TRB Initial Request Form
- TODO (@aterstriep @adamodd?) Updates

## How to test this change

- `scripts/dev up`
- Open the TRB UI
- Create a new TRB request
- Ensure that there's an Attendee in the DB already, even though the FE never called to create one!

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
